installer/scripts/tue-install-impl

```diff
@@ -34,7 +34,7 @@ function _make_sure_installed
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
 # Make sure tools used by this installer are installed
-_make_sure_installed python-yaml git subversion python-pip
+_make_sure_installed python-yaml git subversion python-pip snapd
```

Comment:
Ik had bewust snapd pas geinstalleerd als je ook snaps installeerd. Is best wel een zware dependency anders. Die ook bij elke CI etc aangeroepen wordt.